core/lib/zksync_core/src/consensus/proto/mod.proto

```diff
@@ -0,0 +1,21 @@
+syntax = "proto3";
+
+package zksync.zksync_core.consensus;
```

Comment:
Bikeshedding: Duplicating `zksync` in the package path doesn't look particularly good. Maybe, it makes sense to shorten it to `zksync.core.consensus` or use something like `zksync.server.consensus`?

Reply:
I want the crates to have disjoint proto namespaces, zksync.core.consensus for zksync_core should be ok.